CMakeLists.txt

```diff
@@ -25,6 +25,12 @@ if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "armv7l")
 	set(CMAKE_C_FLAGS "-Ofast -fPIC -mfloat-abi=hard -mfpu=vfp -march=armv7-a -ffast-math -Wfloat-equal -Wshadow -Wcast-align -Wstrict-overflow=5 -Wwrite-strings -Waggregate-return -Wswitch-default -Wswitch-enum -Wformat=2 -g -Wall -Wno-sign-conversion -Wunreachable-code -Wstrict-prototypes")
 endif()
 
+if(${CMAKE_SYSTEM_PROCESSOR} MATCHES "^mips")
+	set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE armhf)
```

Comment:
I know less than zero about Debian packages, I am sure armhf is wrong, but I didn't know what to put there?

Does CMAKE distinguish between mips and mipsel? Can you suggest a better test for the SYSTEM PROCESSOR line?